packages/component/src/Activity/Avatar.js

```diff
@@ -3,22 +3,51 @@ import React from 'react';
 
 import connectToWebChat from '../connectToWebChat';
 
-export default connectToWebChat(
-  ({ styleSet }) => ({ styleSet })
-)(
+const connectAvatar = (...selectors) => connectToWebChat(
   ({
-    children,
-    className,
-    fromUser,
-    styleSet
-  }) =>
+    styleSet: {
+      options: {
+        botAvatarImage,
+        botAvatarInitials,
+        userAvatarImage,
+        userAvatarInitials
+      }
+    }
+  }, { fromUser }) => ({
+    avatarImage: fromUser ? userAvatarImage : botAvatarImage,
+    avatarInitials: fromUser ? userAvatarInitials : botAvatarInitials
+  }),
+  ...selectors
+);
+
+// TODO: [P2] Consider memoizing "style={ backgroundImage }" in our upstreamers
+//       We have 2 different upstreamers <CarouselFilmStrip> and <StackedLayout>
+
+const Avatar = ({
+  avatarImage,
+  avatarInitials,
+  className,
+  fromUser,
+  styleSet
+}) =>
+  !!(avatarImage || avatarInitials) &&
     <div
       className={ classNames(
         styleSet.avatar + '',
         { 'from-user': fromUser },
         (className || '') + ''
       ) }
     >
-      { children }
+      { avatarInitials }
```

Comment:
I'm worried that this won't be the best experience for a screen reader user because the user's initials don't give much context about the intent of this component. Maybe either adding screen reader only text before the initials or adding an aria-label would help here?

Reply:
That sounds great idea. Since this issue is not related to this PR and is in existing code, I create a new issue for this #1499.

Reply:
:+1: